README.md

```diff
@@ -21,7 +21,8 @@ It allows members to request assignments for jobs they have done, that were not
 
 1. Add `juntagrico-assignment-request` to your `requirements.txt`
 2. Add `'juntagrico_assignment_request',` to the `INSTALLED_APPS` in your `settings.py`
-3. Add `path(r'', include('juntagrico_assignment_request.urls')),` at the end of your `urls.py`
+3. Add `re_path(r'', include('juntagrico_assignment_request.urls')),` at the end of your `urls.py`
+4. Apply migrations with `python manage.py migrate`
```

Comment:
This is part of the deployment and the exact deployment process depends on the setup. On the PaaS it would mean pushing the changes and clicking redeploy. So this instruction would be rather confusing in that case.

I would maybe add it as a hint under "redeploy your project": "Don't forget to apply the migrations with ... (the PaaS does this for you)."